components/button/style/index.less

```diff
@@ -142,9 +142,7 @@
   }
 
   &-background-ghost {
-    background: transparent!important;
```

Comment:
In the current implementation, it is always white color text and color btn? But why is it so? Ghost warning btn i can override....